taurus/__init__.py

```diff
@@ -1 +1,9 @@
 """Data concepts library."""
+
+import warnings
+warnings.warn("The taurus module is deprecated in favor of gemd; "
```

Comment:
Here is the new deprecation warning.

Reply:
Should we specify a point at which we'll remove support for the taurus module in imports?